### ISSUE TITLE
main version cannot parse `snforge` output

### ISSUE THREAD

**Issue by amanusk** [software-mansion/cairo-profiler#45]
Running the main on an output of snforge with `--save-trace-data` returns an error:
```
Error: Failed to deserialize call trace

Caused by:
    missing field `contract_address` at line 14 column 3
```

Attached example trace file

[tests::test_token_sender::test_multisend.json](https://github.com/software-mansion/cairo-profiler/files/14505340/tests.test_token_sender.test_multisend.json)

**Comment by piotmag769** [comment]
I see we changed the field name by accident, sorry for that. Anyways the next profiler release prob won't work with older snforge versions anyways since we will be adding `vm_trace` field as a required, so I think this will no longer be relevant

**Comment by piotmag769** [comment]
@amanusk We released [0.2.0](https://github.com/software-mansion/cairo-profiler/releases/tag/v0.2.0) that is compatible with forge 0.19.0, so I think I can close this one
